### ISSUE TITLE
[Urgent] Tray 2.1.2  Java 8 Update 291 - Print HTML Error

### ISSUE THREAD

**Issue by banhang365** [qzind/tray#778]
Hi,
QZTray 2.1.1-2.1.2
**Java 8 Update 291** (QZTray works fine with the old version but got the bugs with this version)
USB ESC/POS Thermal Receipt Printer 80mm
QZ Tray is failed in HTML when using the new Java version (8 Update 291).
Print successfully at the first time
From the second time it was printed out a very **short paragraph** with no content.

<img src="https://user-images.githubusercontent.com/17844532/113091068-556b5c80-9215-11eb-9ae8-b844da2c2211.jpg" width=300px/>

**Comment by tresf** [comment]
Please try with AdoptOpenJDK 11.  It will not conflict with Java 8 and uses a newer copy of the HTML renderer.

**Comment by banhang365** [comment]
> Please try with AdoptOpenJDK 11. It will not conflict with Java 8 and uses a newer copy of the HTML renderer
> 
> You will need to restart QZ Tray after installing AdoptOpenJDK 11.
> 
> You can find out which Java version QZ is using by visiting http://localhost:8182.

I tried using AdoptOpenJDK 11 and it runs fine on Windows 64bit. But I got error on Windows 32bit, Tray can't not print any thing, this is log contents:
```
2021-03-31 18:40:52,549 [ERROR] Failed to print
java.lang.UnsupportedOperationException: Unable to start JavaFX service
```

**Comment by tresf** [comment]
That is correct, QZ Tray is not officially supported on 32-bit.

**Comment by tresf** [comment]
So we're in talks with Gluon about this, it appears something in newer Java FX versions causes this bug.  For now, downgrading to an older Java 8 version should fix it.  You may also be ok by switching the code to use `{ rasterize: true }`.  What's happening is identical to https://github.com/qzind/tray/issues/513#issuecomment-565051824.  Why our code workarounds have completely broken, we're not yet sure, but we're investigating.

AdoptOpenJDK 11 doesn't suffer this bug because it uses an older JavaFX version which we bundle.

**Comment by tresf** [comment]
A temporary workaround seems to be to bump the pulses needed before printing to about 30.

https://github.com/qzind/tray/blob/99d38ce85e3eb3e30e59a743a747b2bfb7d93944/src/qz/printer/action/WebApp.java#L249-L252


```diff
 load(model, (int frames) -> { 
     // Fire on 30th animation frame
+    if(frames <= 29) {
+       return false;
+    }
     try { 
```

**Comment by tresf** [comment]
Patch submitted via https://github.com/qzind/tray/pull/807.  This will be available starting with QZ Tray 2.1.3.
